pylint/checkers/variables.py

```diff
@@ -740,6 +734,41 @@ def _uncertain_nodes_in_except_blocks(found_nodes, node, node_statement):
             uncertain_nodes.append(other_node)
         return uncertain_nodes
 
+    @staticmethod
+    def _uncertain_nodes_in_try_blocks_when_evaluating_except_blocks(
+        found_nodes, node_statement
+    ):
```

Comment:
```suggestion
    def _uncertain_nodes_in_try_blocks_when_evaluating_except_blocks(
        found_nodes: List[nodes.NodeNG], node_statement: ...
    ) -> List[...]:
 ```
 mypy is not strict yet but we're trying to add typing on all new function :)